CLDConfig/CLDReconstruction.py

```diff
@@ -61,6 +62,14 @@
 cellIDSvc.OutputLevel = INFO
 svcList.append(cellIDSvc)
 
+if len(geoservice.detectors) > 1:
+    raise RuntimeError("Too many XML files for the detector path, please only specify the main file!")
+
+sequenceLoader = SequenceLoader(
+    algList,
+    global_vars={"CONFIG": CONFIG, "geoservice": geoservice},
```

Comment:
```suggestion
    # global_vars can be used in sequenceloaded modules without explicit import
    global_vars={"CONFIG": CONFIG, "geoservice": geoservice},
```

yes, this is described in py_utils, but maybe still worth mentioning explicitly once